.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
I don't agree with that blog at all. The official advice from composer is still NOT to include it.